index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio:Pritikant Dhal</title>
+  <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+</head>
+<body>
+  <!--Header Section-->
+  <header class="flex">
+    <img src="images/icons/image_geometry_1.svg" alt="" />
+    <nav class="nav">
+    <svg xmlns="http://www.w3.org/2000/svg" 
+         fill="none" 
+         viewBox="0 0 24 24" 
+         stroke-width="1.5" 
+         stroke="currentColor"
+         class="w-6 h-6"
+         id="menu-1">
+      <path stroke-linecap="round" 
+       stroke-linejoin="round" 
+       d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />   
+    </svg>
+  </nav>  
+  </header>
+
+  <!--Main Content Section-->
+  <section id="main" class="flex popins">
+    <h1 class="title playfair" >Pritikant Dhal</h1>
+    <p class="my-introduction">
+      Hello! I am a software developer! I can help you build a product, 
+      feature or website. Take a look at my work.<br/><br/>
+      If you like what you see and have a project you need coded,
+      don’t hesitate to contact me.
+    </p>
+    <button class="btn">Start collaboration</button>
+    <figure class="main-end">
+      <img src="images/icons/image_geometry_2.svg"
+       alt="main background section" />
+    </figure>
```

Comment:
As stated by the previous reviewer, please make this image a background image instead of using the `<img>` tag. The reason for this is that the image in the design is decorative and should not be in the flow of the page. Also, styling it as a background image will increase your websites performance